### DISCUSSION TITLE
Minnesota - Quick Start - Wrong Version

### DISCUSSION THREAD

**Discussion by Didi211** [edgexfoundry/community#221] [Q&A]
While following quick start, I've encountered problem with running docker containers. The picture shows additional service for MQTT communication from **Export Data** section. On the picture it is written 2.0.0 version, and after I changed to 3.0.0 (which is minnesota), it worked. So I am writing here in hope for edgex developers and website editors to see and fix this typo so the future users won't have the same problem.
![image](https://github.com/edgexfoundry/community/assets/82868612/59d97996-5230-43ee-8b11-b46f41fbc716)
P.S. Maybe this or similar typo  exists on other versions, so it would be nice to fix them all.

**Comment by jiekechoo** [comment]
@Didi211 
Please use the latest version v3.1(Napa), it just released few days ago.
https://docs.edgexfoundry.org/3.1/getting-started/quick-start/#exporting-data

**Comment by lenny-goodell** [comment]
@Didi211, the quick start has been update for the Nap 3.1 release here: https://docs.edgexfoundry.org/3.1/getting-started/quick-start/#exporting-data

Note it is using the nexus3.edgexfoundry.org:10004/app-service-configurable:latest image which is the current dev image. So it should work, but isnt the released image. So the issue need to address that it should be the released image.

Also note since the release on Napa 3.1, it is the current supported release.
